core/src/main/kotlin/authorization/OrtPrincipal.kt

```diff
@@ -49,3 +49,9 @@ fun OrtPrincipal?.hasRole(role: String) = this != null && role in roles
  * Return true if this [OrtPrincipal] is not `null` and has the [superuser role][Superuser.ROLE_NAME].
  */
 fun OrtPrincipal?.isSuperuser() = hasRole(Superuser.ROLE_NAME)
+
+fun OrtPrincipal.getUserId(): String = payload.subject
```

Comment:
Nit: Not sure why these are extension functions. For the other ones, it makes sense, since they operate on a nullable target. But the new ones can as well be regular methods of the class, or even better properties.